### PR TITLE
Isuue #PS-2909 & PS-2907   : Show subtopics for topics which has same name in the course-planner and multi-select options for topic 

### PR DESCRIPTION
**Pull request by suvarnakale**

### CODE REVIEW

File: src/pages/centers/[cohortId]/index.tsx [hunk 1]
```diff
@@ -212,8 +212,36 @@ const CohortPage = () => {
...
+    if (cohortId) {
+      const filters = {
+        cohortId,
+        role: Role.STUDENT,
+        // status: [Status.ACTIVE],
+      };
 
+      try {
+        const response = await getMyCohortMemberList({
+          // limit: 20,
```
Comment (itsvick) [inline src/pages/centers/[cohortId]/index.tsx:224]: @suvarnakale i think this code has been removed by @shreyas1434shinde please check once and remove it, if not required

**Review by itsvick — CHANGES_REQUESTED** [review]

File: src/components/TopicDetails.tsx [hunk 2]
```diff
@@ -74,7 +74,9 @@ const TopicDetails: React.FC<TopicDetailsProps> = ({
         if (filteredData[0].resourceType === 'Course') {
           router.push(`/course-hierarchy/${filteredData[0].identifier}`);
         } else {
-          router.push(`/play/content/${filteredData?.[0]?.identifier?.toLowerCase()}`);
+          router.push(
```
Comment (itsvick) [inline src/components/TopicDetails.tsx:77]: @suvarnakale As `filteredData?.[0]?.identifier` used in both the cases can u add one more if condition on top of it... for additional check... u can remove `?.` operator from below statements then

File: src/pages/centers/[cohortId]/events/[date]/index.tsx [hunk 3]
```diff
@@ -97,7 +97,7 @@ const EventMonthView: React.FC<any> = () => {
 
           if (response?.events.length > 0) {
             response?.events.forEach((event: any) => {
-              if (event.isRecurring) {
+              if (event?.metadata?.type === 'planned') {
```
Comment (itsvick) [inline src/pages/centers/[cohortId]/events/[date]/index.tsx:100]: @suvarnakale can u add `planned` and `extra`, session in enum... apply it everywhere wherever applicable

File: src/pages/centers/[cohortId]/index.tsx (hunk 1 above)
Comment (suvarnakale) [reply src/pages/centers/[cohortId]/index.tsx:224]: I have not added it newly.. Its already there.

File: src/components/TopicDetails.tsx (hunk 2 above)
Comment (suvarnakale) [reply src/components/TopicDetails.tsx:77]: sure

File: src/pages/centers/[cohortId]/events/[date]/index.tsx (hunk 3 above)
Comment (suvarnakale) [reply src/pages/centers/[cohortId]/events/[date]/index.tsx:100]: sure.

**Pull request merged**